Sources/IO/XML/XMLReader/index.js

```diff
@@ -18,6 +18,48 @@ function stringToXML(xmlStr) {
   return (new DOMParser()).parseFromString(xmlStr, 'application/xml');
 }
 
+/**
+ * Extracts binary data out of a file bytearray given a prefix/suffix.
+ */
+function extractBinary(arrayBuffer, prefixRegex, suffixRegex = null) {
```

Comment:
That function can be used for other parser, therefore, we may want it in a more generic spot or we need to export it in some way...

Reply:
I was thinking somewhere in vtk-js/Utilities since it is generic enough to come out of the XML directory. Not sure if it'll be used outside of IO though, so maybe some IO utilities?

Reply:
IO/Core/BinaryHelper

Reply:
Maybe with a set of functions like:

```
uint64ToUint32(...)
binaryToString(arrayBuffer) => str
...
```

Reply:
I've added an initial IO/Core/BinaryHelper.